server/lib/config/pbench-server-default.cfg

```diff
@@ -29,6 +29,28 @@ mailto=%(admin-email)s
 mailfrom=%(user)s@%(host)s
 commit_id=unknown
 
+# Configuration to support remote reverse-proxy configuration (e.g., NGINX)
+# in front of Pbench server.
+#
+# The proxy_host should be specified using the <host>:<port> to which
+# clients are expected to connect to Pbench through the reverse proxy
+# server. For example, "pbench.example.com:8901".
+#
+# This is advertised to front-end clients asking for endpoint configuration.
+# By default, this is the http port (80) on the server host, which the default
+# Pbench Ansible role pbench-server-activate-httpd-setup-restart will configure
+# as a local reverse-proxy to the API %(bind_port).
+#
+# To configure Pbench to advertise a remote reverse-proxy, set "proxy_host=host:port"
+# in pbench-server.cfg. Note that the local %(bind_port) must then be opened in
+# firewalld, and the reverse proxy must pass %(rest_uri) traffic on the proxy
+# port to %(host):%(bind_port) and other traffic to %(host):80. For example,
+# NGINX should have a "location /api" with at least "proxy_pass http://<host>:<bind_port>"
+# and a "location /" with at least "proxy_pass http://<host>:80" if the Pbench host
+# Apache isn't configured to listen on a non-default port. (Further instruction
+# in reverse proxy configuration is beyond the scope of this mega-comment.)
+proxy_host=%(host)s
```

Comment:
I think we can drop this entirely.

The reverse proxy is configured to pass along in a header the original URL that it accepted the request on, so that the backend server, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host.

Reply:
Sounds good to me.

Reply:
Interestingly, there appears to be a newer `Forwarded` header (with a somewhat more obtuse syntax), combining the non-standard `X-Forwarded-For`, `X-Forwarded-Host`, and `X-Forwarded-Proto`.

Maybe I'll try to support either. I'll probably have to try to set up a remote reverse-proxy to my VM to test it. Maybe we can set up another port on perf44??

Reply:
> Maybe I'll try to support either. I'll probably have to try to set up a remote reverse-proxy to my VM to test it. Maybe we can set up another port on perf44??

And ... of course I don't need that because I can fake any headers I want through Postman.